addons/lunch/static/src/js/lunch.js

```diff
@@ -0,0 +1,44 @@
+openerp.lunch  = function(instance) {
```

Comment:
Declare your widget class and assign it to the `openerp.lunch` in the function closure : 
`(function(){ /* your code*/ })();`, like it is done in im_chat.js, chrome.js, ...